src/OOPMacroImpl.jl

```diff
@@ -115,7 +115,7 @@ macro class(ClsName, Cbody)
               end
               """
     # Escape here because we want ClsName and the methods be defined in user scope instead of OOPMacro module scope.
```

Comment:
The comment should be updated. Particullary, "user scope" should be change to "the module issuing the @class macro" or something like that.

Reply:
done

Reply:
Sorry, `@Class` shold be lower case `@class`.

Reply:
cool, done :)